RELEASE.md

```diff
@@ -0,0 +1,123 @@
+# Release instructions
+
+In order to release a new version of Apache Baremaps, follow these steps:
+
+- [ ] Notify the mailing list and ask everyone to pause commits on the main branch
+- [ ] Create a new issue on GitHub with the title "Release Baremaps <version>"
+- [ ] Create a new branch for the release (e.g. `release-<version>`)
+
+```bash
+git checkout -b release-<version>
+git push --set-upstream origin release-<version>
+```
+
+- [ ] Set the release version and commit the changes:
+
+```bash 
+mvn versions:set -DnewVersion=<version>
+git commit -a -m "Release Baremaps <version>"
+```
+
+- [ ] Tag the last commit with the release candidate version:
+
+```bash
+git tag v<version>-rc<number>
+```
+
+- [ ] Push the tag to the remote repository:
+
+```bash
+git push origin v<version>-rc<number>
+```
+
+- [ ] Generate the release notes for this tag on GitHub.
+- [ ] Generate the artifacts:
+
+```bash
+./scripts/generate-artifacts.sh
+```
+
+- [ ] Publish the artifacts:
+
+```bash
+svn co https://dist.apache.org/repos/dist/dev/incubator/baremaps/ baremaps-dev
+mkdir baremaps-dev/<version>-rc<number>
+cp ./baremaps-cli/target/baremaps-<version>-incubating-* baremaps-dev/<version>-rc<number>/.
+svn commit -m "Baremaps <version>-rc<number>"
+```
+
+- [ ] Ask the community to vote for the release candidate.
+- [ ] If the release candidate is not approved by the community, commit the necessary changes, clean the git history, and go back to step 5.
+- [ ] If the release candidate is approved by the community, tag the release commit with the release version:
+
+```bash
+git tag -a v[version]
+git push origin v[version]
+```
+
+- [ ] Publish the artifacts:
+
+```bash
+svn co https://dist.apache.org/repos/dist/release/incubator/baremaps/ baremaps-release
+mkdir baremaps-release/<version>
+cp ./baremaps-cli/target/baremaps-<version>-incubating-* baremaps-release/<version>/.
```

Comment:
@bchapuis is the documentation here missing information about how to add the files that Julien highlight missing in the release ?

Cf: #604